style.css

```diff
@@ -1,403 +1,1091 @@
+  gap: 142px;
+  align-items: center;
+  background-color: #f7f7f7;
+  justify-content: center;
+  padding-top: 100px;
+  padding-bottom: 100px;
+}
+
+.work {
+  margin-left: 142px;
+  margin-right: 142px;
+  padding: 24px;
+  border: gray solid 0.3px;
+  border-radius: 20px;
+  background: white;
+  display: flex;
+}
+
+.work-subtitle {
+  display: flex;
+  gap: 12px;
+  align-items: center;
+}
+
+.work-title {
+  font-weight: 600;
+  font-size: 3em;
+  color: black;
+}
+
+.work-details {
+  align-items: center;
+  padding-left: 46px;
+  padding-right: 46px;
+}
+
+.tags {
+  display: flex;
+  gap: 12px;
+  margin-bottom: 20px;
+}
+
+.tag {
+  color: #6170ff;
+  background-color: #ebebff;
+  padding: 10px;
+  border-radius: 10px;
+}
+
+.company {
+  text-transform: uppercase;
+  font-weight: bold;
+}
+
+.work-link {
+  color: #6071fe;
+  text-decoration: none;
+  margin-top: 20px;
+  font-size: medium;
+  border: #6071fe solid 1px;
+  border-radius: 10px;
+  padding: 10px;
+}
+
+.work-link:hover {
+  background-color: #6071fe;
+  color: white;
+}
+
+.work-link:active {
+  background: #2230d2;
+  color: white;
+}
+
+.about-section {
+  display: flex;
+  padding: 142px;
+  background: white;
+  border-radius: 0 100px 0 0;
+}
+
+.about {
+  width: 42%;
+  margin-right: 10px;
+}
+
+.languages {
+  width: 57%;
+  display: block;
+}
+
+hr {
+  height: 0.2px;
+  border: 0.2px solid gray;
+  width: 100%;
+}
+
+.about-title {
+  font-weight: 600;
+  font-size: 3em;
+  color: black;
+}
+
+.languages-title {
+  font-weight: 600;
+  font-size: 1.2em;
+  color: #344563;
+  margin-top: 20px;
+}
+
+.languages-head {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  cursor: pointer;
+}
+
+.language-container {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 12px;
+  margin-top: 20px;
+  margin-bottom: 36px;
+}
+
+.language-card {
+  padding: 20px;
+  width: 100px;
+  border-radius: 10px;
+  background: #f7f7f7;
+}
+
+.frameworks,
+.skills {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 12px;
+  margin-top: 20px;
+  margin-bottom: 36px;
+}
+
+.about-work-link {
+  color: #6071fe;
+  text-decoration: none;
+  margin-top: 20px;
+  font-size: medium;
+  border: #6071fe solid 1px;
+  border-radius: 10px;
+  padding: 10px;
+  width: 40%;
+}
+
+.about-work-link:hover {
+  background-color: #6071fe;
+  color: white;
+}
+
+.about-work-link:active {
+  background: #2230d2;
+  color: white;
+}
+
+.text {
+  display: flex;
+  justify-content: space-between;
+  padding-top: 24px;
+  padding-bottom: 24px;
+}
+
+.contact {
+  display: flex;
+  flex-direction: column;
+  background: #6071fe;
+  justify-content: center;
+  align-items: center;
+  padding: 24px;
+  border-radius: 50px 0 0;
+}
+
+.contact-title {
+  font-weight: 600;
+  font-size: 2em;
+  color: white;
+}
+
+.contact-subtitle {
+  font-weight: 100;
+  font-size: 1.2em;
+  text-align: center;
+  color: #d9dbff;
+  padding-top: 10px;
+  padding-bottom: 10px;
+}
```

Comment:
- [ ] **[OPTIONAL]** Please 🙏🏽 add padding-left and padding-right to look similar to the Figma template: 👇🏽 
![image](https://user-images.githubusercontent.com/58642949/187027889-43700cc8-91a1-426c-8ab7-0ca847aeaa20.png)

Reply:
Hello, Please review, I have added padding-left and padding-right. Given that this review has a [OPTIONAL] prefix, is my project going to be approved?

Reply:
Hello @franclobo, Please review, I have added padding-left and padding-right. Given that this review has a [OPTIONAL] prefix, is my project going to be approved?